### DISCUSSION TITLE
Actions for selected containers

### DISCUSSION THREAD

**Discussion by shinjiikari256** [podman-desktop/podman-desktop#2981] [Ideas]
How difficult is it to add actions for all selected containers?
And for groups without the need for selection?

I found [this issue](https://github.com/containers/podman-desktop/issues/333) and it is closed.
But I don't notice this in the interface:
![image](https://github.com/containers/podman-desktop/assets/39000784/c9a16a56-c120-4b2b-8c96-cea87a3a2584)

**Comment by benoitf** [comment]
hello @shinjiikari256 when you select containers on the left, bulk actions are 'delete' and 'podify' (the delete icon and cube icons with 'On 3 selected items' on the right of the search containers widget

![image](https://github.com/containers/podman-desktop/assets/436777/785aa18e-ac0b-4f7f-b32e-1061189e40d3)

**Reply by shinjiikari256** [reply]
Oh, yes, them I see it.
But it would be useful to also start and stop the selected containers?

**Reply by benoitf** [reply]
you can file enhancement request at https://github.com/containers/podman-desktop/issues/new?assignees=&labels=kind%2Fenhancement&projects=&template=enhancement.yml
